test/integ/conftest.py

```diff
@@ -153,21 +156,8 @@ def init_test_schema(request, db_parameters) -> None:
         protocol=ret["protocol"],
     ) as con:
         con.cursor().execute("CREATE SCHEMA IF NOT EXISTS {}".format(TEST_SCHEMA))
-
-    def fin():
-        ret1 = db_parameters
-        with snowflake.connector.connect(
-            user=ret1["user"],
-            password=ret1["password"],
-            host=ret1["host"],
-            port=ret1["port"],
-            database=ret1["database"],
-            account=ret1["account"],
-            protocol=ret1["protocol"],
-        ) as con1:
-            con1.cursor().execute("DROP SCHEMA IF EXISTS {}".format(TEST_SCHEMA))
-
-    request.addfinalizer(fin)
```

Comment:
Why get rid of the finalizer method? Just curious because I copied this piece of code to snowpark-python and wondering if I should also make this change here

Reply:
It's because the new way of doing this is to yield what the fixture would have returned before and everything after the yield will be executed at tear-down time, see documentation here: https://docs.pytest.org/en/6.2.x/fixture.html#yield-fixtures-recommended